app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Yajra\Datatables\Datatables;
+use App\Models\User;
+use App\Http\Resources\User as UserResource;
+
+class UserController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     *
+     */
+    public function index() {
+
+        return view('admins/user');
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 22</br>Opening brace should be on a new line